typescript/sdk/src/token/configUtils.ts

```diff
@@ -148,17 +145,19 @@ export async function expandWarpDeployConfig(params: {
       ...config,
     };
 
-    // Virtual Config Expansion
-    if (includeVirtual) {
-      chainConfig.contractVerificationStatus = {
-        [hypERC20contracts[config.type]]: true,
-      };
+    // Expand warpDeployConfig virtual to the control states
+    if (expandedOnChainWarpConfig?.[chain].contractVerificationStatus) {
+      // For most cases, we set to Verified
+      chainConfig.contractVerificationStatus = objMap(
+        expandedOnChainWarpConfig[chain].contractVerificationStatus ?? {},
+        (_, status) => {
+          // Skipped for local e2e testing
+          if (status === ContractVerificationStatus.Skipped)
+            return ContractVerificationStatus.Skipped;
```

Comment:
Can we add a condition here to skip if the protocol type is not a supported one like SVM etc?

Reply:
yup